Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -100,6 +97,29 @@ extension TuistCore.Target {
                                 environment: environment,
                                 launchArguments: launchArguments,
                                 filesGroup: .group(name: "Project"),
-                                dependencies: dependencies)
+                                dependencies: dependencies,
+                                playgrounds: playgrounds)
+    }
+
+    fileprivate static func sourcesAndPlaygrounds(manifest: ProjectDescription.Target, targetName: String, generatorPaths: GeneratorPaths) throws -> (sources: [TuistCore.Target.SourceFile], playgrounds: [AbsolutePath]) {
+        var sourcesWithoutPlaygrounds: [(path: AbsolutePath, compilerFlags: String?)] = []
+        var playgrounds: Set<AbsolutePath> = []
+
+        // Sources
+        let allSources = try TuistCore.Target.sources(targetName: targetName, sources: manifest.sources?.globs.map { (glob: ProjectDescription.SourceFileGlob) in
+            let globPath = try generatorPaths.resolve(path: glob.glob).pathString
+            let excluding: [String] = try glob.excluding.compactMap { try generatorPaths.resolve(path: $0).pathString }
+            return TuistCore.SourceFileGlob(glob: globPath, excluding: excluding, compilerFlags: glob.compilerFlags)
+        } ?? [])
+
+        allSources.forEach { path, compilerFlags in
+            if path.pathString.range(of: "\\.playground/Contents\\.swift$", options: .regularExpression) == nil {
```

Comment:
There could be other sources in a playground directory. Contents.swift is a well-known path, but just want to make sure that it would still support adding new pages, and supporting Swift files inside the .playground bundle.

Reply:
That's a good point, I think I can make the regex more generic: `\.playground/.+`